docker-compose-openelis.yml

```diff
@@ -0,0 +1,234 @@
+      # Env variables passed to the common properties file
+      - SSL_KEYSTORE_PATH
+      - SSL_KEYSTORE_PASSWORD
+      - SSL_TRUSTSTORE_PATH
+      - SSL_TRUSTSTORE_PASSWORD
+    volumes:
+      -  key_trust-store-volume:/etc/openelis-global
+      - ../../distro/binaries/openelis/volumes/plugins/:/var/lib/openelis-global/plugins
+      - ../../distro/binaries/openelis/volumes/logs/oeLogs:/var/lib/openelis-global/logs
+      - ../../distro/binaries/openelis/volumes/logs/tomcatLogs/:/usr/local/tomcat/logs
+      - ../../distro/binaries/openelis/volumes/properties/SystemConfiguration.properties:/var/lib/openelis-global/properties/SystemConfiguration.properties
+      - ../../distro/binaries/openelis/volumes/analyzer/analyzer-test-map.csv:/var/lib/openelis-global/analyzer/analyzer-test-map.csv
+      - ../../distro/binaries/openelis/volumes/odoo/odoo-test-product-mapping.csv:/var/lib/openelis-global/odoo/odoo-test-product-mapping.csv
+      - ../../distro/binaries/openelis/volumes/ocl:/var/lib/openelis-global/ocl
+      -  lucene_index-vol:/var/lib/lucene_index
+    secrets:
+      - source: common.properties
+
+  fhir.openelis.org:
+    container_name: external-fhir-api
+    image: itechuw/openelis-global-2-fhir:develop
+    platform: linux/amd64
+    ports:
+      - "8086:8080"
+      - "8444:8443"
+    depends_on:
+      - db.openelis.org
+      - certs
+    networks:
+      - openelis-network
+    restart: always
+    environment:
+      TZ: America/New_York
+
+      JAVA_OPTS: "-Djavax.net.ssl.trustStore=${SSL_TRUSTSTORE_PATH}
+                      -Djavax.net.ssl.trustStorePassword=${SSL_TRUSTSTORE_PASSWORD}
+                      -Djavax.net.ssl.trustStoreType=pkcs12
+                      -Djavax.net.ssl.keyStore=${SSL_KEYSTORE_PATH}
+                      -Djavax.net.ssl.keyStorePassword=${SSL_KEYSTORE_PASSWORD}
+                      -Djavax.net.ssl.keyStoreType=pkcs12"
+      # Config variables loaded through Tomacat server.xml
+      CATALINA_OPTS: "-Dhapi.ssl.truststorepath=${SSL_TRUSTSTORE_PATH} -Dhapi.ssl.truststorepassword=${SSL_TRUSTSTORE_PASSWORD} -Dhapi.ssl.keystorepath=${SSL_KEYSTORE_PATH} -Dhapi.ssl.keystorepassword=${SSL_KEYSTORE_PASSWORD}"
+      # Config variables loaded through Hapi application.yml
+      FHIR_DATASOURCE_URL : "jdbc:postgresql://db.openelis.org:5432/clinlims?currentSchema=clinlims"
+      FHIR_DATASOURCE_USERNAME: "clinlims"
+      FHIR_DATASOURCE_PASSWORD: ${OE_DB_PASSWORD}
+      FHIR_SERVER_ADRESS: "http://fhir.openelis.org:8080/fhir/"
+
+    logging: *local-logging
+    volumes:
+      -  key_trust-store-volume:/etc/openelis-global
+
+  frontend.openelis.org:
+    image: itechuw/openelis-global-2-frontend:develop
+    container_name: openelisglobal-front-end
+    platform: linux/amd64
+    networks:
+      - openelis-network
+    environment:
+      - CHOKIDAR_USEPOLLING=true
+    logging: *local-logging
+    tty: true
+
+  openelis.proxy:
```

Comment:
Is there anything that makes this proxy special, or can we make it work with our proxy?